publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/helper/RabbitMqProperties.java

```diff
@@ -492,8 +494,13 @@ public void shutdownCompleted(ShutdownSignalException cause) {
                 }
             });
             BasicProperties msgProps = MessageProperties.BASIC;
+            final BasicProperties PERSISTENT_BASIC_APPLICAION =
```

Comment:
Initialization of PERSISTENT_BASIC_APPLICATION should be stored in MessageProperties class, next to intialization of other properties, e.g. MINIMAL_BASIC, BASIC, etc. Initialization done here causes that a new instance of the BasicProperties is created every time send() method is invoked.

Reply:
I didn't notice the MessageProperties class is part of rabbitmq client library:-( Initialization of the properties should be done at class level of RabbitMqProperties.

Reply:
Made the proper changes